WebServer/microservices/video-download/unittest/video_file_saver_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest2
+import sys
+import os
+sys.path.append(os.getcwd() + '/..')
+import video_file_saver
+
+class AuthTokenTests(unittest2.TestCase):
+    @classmethod
+    def tearDown(cls):
+        [os.remove(f) for f in os.listdir() if f.endswith('.mp4') or f.endswith('.txt')]
+        
+    def test_record(self):
+        stream = 'https://cph-p2p-msl.akamaized.net/hls/live/2000341/test/master.m3u8'
```

Comment:
Kræver det en internetforbindelse for, at test passer?

Reply:
formentlig